accounting/accounting_cli.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import sqlite3
+import argparse
+import time
+
+import pandas as pd
+
+from accounting import accounting_cli_functions as aclif
+
+
+def main():
+
+    conn = sqlite3.connect("FluxAccounting.db")
```

Comment:
Nit pick, but I would put this connection establishment after the command line parsing, so that a connection is not made when a user provides bad input.

Reply:
Thanks for pointing this out! I will make this change right now and push an updated branch.